lib/wibox/init.lua

```diff
@@ -8,14 +8,16 @@
 local capi = {
     drawin = drawin,
     root = root,
-    awesome = awesome
+    awesome = awesome,
+    screen = screen
 }
 local setmetatable = setmetatable
 local pairs = pairs
 local type = type
 local object = require("gears.object")
 local beautiful = require("beautiful")
 local base = require("wibox.widget.base")
+local ascreen = require("awful.screen")
```

Comment:
I'm afraid that `wibox` shouldn't depend on functions from `awful`. We already have enough "weird dependency loops" (My hack around this was to have a copy called `screen_getbycoord` in `lib/wibox/drawable.lua`). Perhaps this should be worked around by moving some helper function into `gears`?